clouddriver-cloudrun/src/main/java/com/netflix/spinnaker/clouddriver/cloudrun/model/CloudrunSpec.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2023 OpsMx, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.cloudrun.model;
+
+public class CloudrunSpec {
+
+  private CloudrunSpecTemplate template;
+
+  private CloudrunSpecTraffic[] traffic;
+
+  public CloudrunSpecTemplate getTemplate() {
+    return template;
+  }
+
+  public void setTemplate(CloudrunSpecTemplate template) {
+    this.template = template;
+  }
+
+  public CloudrunSpecTraffic[] getTraffic() {
+    return traffic;
+  }
+
+  public void setTraffic(CloudrunSpecTraffic[] traffic) {
+    this.traffic = traffic;
+  }
```

Comment:
Same comment wrt Lombok

Reply:
@mattgogerly Please can you review the changes now?

Reply:
@mattgogerly Please can you review the changes?

Reply:
@mattgogerly Please can you review the changes?

Reply:
@mattgogerly Please can you review the changes?